src/main/java/org/jenkinsci/plugins/gitclient/CliGitAPIImpl.java

```diff
@@ -800,7 +800,7 @@ public void submoduleClean(boolean recursive) throws GitException, InterruptedEx
     	if (recursive) {
             args.add("--recursive");
     	}
-    	args.add("git clean -fdx");
+        args.add("git clean -ffdx");
```

Comment:
I recommend against submitting a separate commit to cleanup indentation.  I'm afraid I'll just reject an indentation cleanup pull request.  That's not how this particular project chooses to operate.

Early in my submissions to the plugin I proposed that we switch the plugin to use an automatic formatting step in the maven build process. I've used automatic formatting on projects at work for years and have been very happy with the diff minimization and format standardization which results.  The other developers on the project were pretty strongly opposed to the automatic formatting idea (creates a "diff wall", damages the layout of calls to fluent interfaces, etc.)

I've accepted that, for this project, I will minimize white space changes from one commit to the next, and will ignore the inconsistencies which exist within the existing source base.